Orchestrator.py

```diff
@@ -747,4 +783,151 @@ def _computeHeadingSpeedMovementTimeout(self, dotBotId, path):
             movementTimeout)
         )
 
-        return (heading, speed, movementTimeout)
\ No newline at end of file
+        return (heading, speed, movementTimeout)
+
+    # === Relays
+
+    def _assignRelaysAndRelayPositionsCb(self):
+
+        log.debug('heartbeats {}'.format([db['heartbeat'] for (_, db) in self.dotBotsView.items()]))
+        self.simEngine.schedule(self.simEngine.currentTime() + 10, self._assignRelaysAndRelayPositionsCb)
+
+        if self.relayAlgorithm   == "Recovery":
+            self._recoveryRelayPlacement()
```

Comment:
turn name around so we have the same prefix: `_relayPlacementRecovery()` and `_relayPlacementSelfHealing()`